PhysicsTools/NanoAOD/python/egamma_custom_cff.py

```diff
@@ -7,14 +7,88 @@
 from PhysicsTools.NanoAOD.nanoDQM_cff import _Photon_extra_plots, _Electron_extra_plots
+from PhysicsTools.NanoAOD.triggerObjects_cff import triggerObjectTable, mksel
+
+customElectronFilterBits = cms.PSet(
+    doc = cms.string("PixelMatched e/gamma"), # this may also select photons!
+    id = cms.int32(11),
+    sel = cms.string("type(92) && pt > 7 && (coll('hltEgammaCandidates') || coll('hltEgammaCandidatesUnseeded')) && (filter('*PixelMatchFilter') || filter('*PixelMatchUnseededFilter'))"),
+    l1seed = cms.string("type(-98)"),  l1deltaR = cms.double(0.3),
+    skipObjectsNotPassingQualityBits = cms.bool(True),
+    qualityBits = cms.VPSet(
+        #HLT_Ele30_WPTight_Gsf
+        mksel("filter('hltEGL1SingleEGOrFilter')","1e WPTight L1T match"), 
+        mksel("filter('hltEG30L1SingleEGOrEtFilter')","1e WPTight Et"),
+        mksel("filter('hltEle30WPTightClusterShapeFilter')","1e WPTight SigmaIeIe"),
+        mksel("filter('hltEle30WPTightHEFilter')","1e WPTight HoE"),
+        mksel("filter('hltEle30WPTightEcalIsoFilter')","1e WPTight ECAL Iso"),
+        mksel("filter('hltEle30WPTightHcalIsoFilter')","1e WPTight HCAL Iso"),
+        mksel("filter('hltEle30WPTightPixelMatchFilter')","1e WPTight Pixel match"),
+        mksel("filter('hltEle30WPTightPMS2Filter')","1e WPTight S2"),
+        mksel("filter('hltEle30WPTightGsfOneOEMinusOneOPFilter')","1e WPTight 1/E-1/p"),
+        mksel("filter('hltEle30WPTightGsfMissingHitsFilter')","1e WPTight missing hits"),
+        mksel("filter('hltEle30WPTightGsfDetaFilter')","1e WPTight DEta"),
+        mksel("filter('hltEle30WPTightGsfDphiFilter')","1e WPTight DPhi"),
+        mksel("filter('hltEle30WPTightGsfTrackIsoFilter')","1e WPTight Track Iso"),
+        #HLT_Ele23_Ele12_CaloIdL_TrackIdL_IsoVL
+        mksel("filter('hltEGL1SingleAndDoubleEGOrPairFilter')","2e L1T match"), 
+        mksel("filter('hltEle23Ele12CaloIdLTrackIdLIsoVLTrackIsoLeg1Filter')","2e Track Iso Leg1"),
+        mksel("filter('hltEle23Ele12CaloIdLTrackIdLIsoVLTrackIsoLeg2Filter')","2e Track Iso Leg2"),
+        #HLT_DoubleEle33_CaloIdL_MW
+        mksel("filter('hltEGL1SingleAndDoubleEGNonIsoOrWithEG26WithJetAndTauFilter')","2e (CaloIdL_MW) L1T match"), 
+        mksel("filter('hltEle33CaloIdLPixelMatchFilter')","2e (CaloIdL_MW) Pixel match Leg1"),
+        mksel("filter('hltEle33CaloIdLMWPMS2Filter')","2e (CaloIdL_MW) S2"),
+        mksel("filter('hltDiEle33CaloIdLMWPMS2UnseededFilter')","2e (CaloIdL_MW unseeded) S2"),
+        #HLT_Photon200
+        mksel("filter('hltEG200HEFilter')","1e Photon200"),
+        #HLT_Photon50EB
+        mksel("filter('hltEG50EBEtFilter')","1e Photon50EB"),
+        #HLT_Ele16_Ele12_Ele8_CaloIdL_TrackIdL
+        mksel("filter('hltEle16Ele12Ele8CaloIdLTrackIdLDphiLeg3Filter')","3e Leg3"),
+        #HLT_Mu12_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL
+        mksel("filter('hltMu12TrkIsoVVLEle23CaloIdLTrackIdLIsoVLElectronlegTrackIsoFilter')","1mu-1e eLeg"),
```

Comment:
Note: this might change to `hltMuTrkIsoVVLEle23CaloIdLTrackIdLIsoVLElectronlegTrackIsoFilter` as part of [CMSHLT-2984](https://its.cern.ch/jira/browse/CMSHLT-2984). Using wildcards as in [*] is probably wiser.

[*] https://github.com/cms-sw/cmssw/blob/3db814726516bc6276ef0fd623d89c9250d0b75e/PhysicsTools/NanoAOD/python/triggerObjects_cff.py#L68

Reply:
Thanks @missirol, I didn't know about this. 
What we want to achieve with these new filter bits in EGM Nano is exactly to get rid of wildcards, so that in principle  we have a perfect control of what we are measuring (if, e.g., we are using Nano for efficiencies) and no contamination from other paths or from similar paths with different thresholds. For this specific case and since we are talking about an electron leg that is the same for several mu-ele paths, we don't have the aforementioned problem and I am OK to add one wildcard as follows:
`hltMu*TrkIsoVVLEle23CaloIdLTrackIdLIsoVLElectronlegTrackIsoFilter`
to be prepared for the possible change in the naming convention in the menu. I will have to change that in `cms-sw:master` too, since there the new version with no wildcards was already merged.

Reply:
Okay, thanks @RSalvatico.

Reply:
@RSalvatico It seems that we lost track of propagating this change to `master`. Could you please open a PR? Thanks!

Reply:
@hqucms Yes, done in #44809.